velox/serializers/SparkSerializer.cpp

```diff
@@ -0,0 +1,761 @@
+/*
```

Comment:
Is there any other proper name instead of `SparkXXX`?

Reply:
Would `RowVectorSerializer` make sense?

Reply:
The super class is following, it is really `RowVectorSerializer`, other serializer can append several `RowVectorPtr`, but this serializer can only append one non-empty vector, so I named it `SingleVectorSerializer`
```
class VectorSerializer {
 public:
  virtual ~VectorSerializer() = default;

  virtual void append(
      const RowVectorPtr& vector,
      const folly::Range<const IndexRange*>& ranges) = 0;

  // Writes the contents to 'stream' in wire format
  virtual void flush(OutputStream* stream) = 0;
};
```

Reply:
Makes sense to me.